### ISSUE TITLE
[Bug]: Fluent UI 9 Popover aria-hidden-focus error

### ISSUE THREAD

**Issue by hemarina** [microsoft/fluentui#30670]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
aria-hidden-focus: Ensures aria-hidden elements are not focusable nor contain focusable elements. Focusable content should have tabindex="-1" or be removed from the DOM.
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://react.fluentui.dev/?path=/docs/components-popover--default

## Actual Behavior
Open link, pick the `With Arrow` example. Click on popover trigger button. In developer mode, check i tag. 
The error message is about the following code when PopoverTrigger Button is not clicked. This is a bug because button is focusable with keyboard tab/mouse. This `<i>` tag is not related to the button. Could we consider remove i tag?

```html
<i tabindex="0" role="none" data-tabster-dummy="" aria-hidden="true" style="position: fixed; height: 1px; width: 1px; opacity: 0.001; z-index: -1; content-visibility: hidden; top: 0px; left: 0px;"></i>
```

## Expected Behavior
Click on popover trigger button. i tag not exist or keep tabindex = 1.

### Requested priority

Normal

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by miroslavstastny** [comment]
The `i` elements are intentional, see the explanation here: https://github.com/microsoft/fluentui/issues/25133#issuecomment-1279196099.
